push-notifications/ios/Plugin/PushNotificationsDelegate.swift

```diff
@@ -0,0 +1,91 @@
+import Capacitor
+import UserNotifications
+
+public class PushNotificationsDelegate: NSObject, NotificationHandlerProtocol {
+    public var plugin: CAPPlugin?
```

Comment:
This creates a retain cycle that will cause problems.
```suggestion
    public weak var plugin: CAPPlugin?
```